package.json

```diff
@@ -8,6 +8,9 @@
     "node": ">=12"
   },
   "scripts": {
+    "build": "npm run build:gitignore && npm run build:ts",
+    "build:ts": "tsc",
+    "build:gitignore": "./scripts/update_gitignore_for_tsc_output.sh run",
```

Comment:
this is fairly hacky and also is not OS-independent. would it be possible to replace it with a JS script?

Reply:
Done!